### ISSUE TITLE
Add code to import list of local schools

### ISSUE THREAD

**Issue by ldodds** [BathHacked/energy-sparks#52]
Add code + rake task to import a list of local schools from a CSV file.

* [ ] Add a unique, external key for the School model to avoid duplicating data or relying on name. Suggest adding a URN field to school which is the unique id for school in Edubase
* [ ] Support all attributes in a CSV file, e.g. name, type, website, address, etc
* [ ] Imported schools should not be enrolled by default (#49)
* [ ] Code shouldn't change any data for existing, enrolled schools

Basically should be safe to run/re-run this to add new schools or update details of non-enrolled schools without clobbering existing setup.

**Comment by ldodds** [comment]
The format for the CSV file should be:

* URN (unique key)
* Name
* Type (primary, secondary)
* Address
* Postcode
* Website
* Ecoschool Status (bronze, silver, green)

Sample data:

```
"URN","Name","Type","Address","Postcode","Website","Ecoschool Status"
109153,"St Saviour’s CofE Junior School","primary","Brookleaze Place, Larkhall, Bath","BA1 6RB","http://www.stsaviours-jun.co.uk/",
109154,"St Saviour’s CofE Infant School","primary","Spring Lane, Larkhall, Bath","BA1 6NY","http://www.stsaviours-infants.org/website","green"
```

Also attached as a zipped CSV.

[school-sample.csv.zip](https://github.com/BathHacked/energy-sparks/files/543961/school-sample.csv.zip)
